docs/user-guide.md

```diff
@@ -0,0 +1,856 @@
+⟺ [JavaScript](https://pyodide.org/en/stable/usage/api/js-api.html)
+[foreign function interface](#ffi) (FFI). This
+bridges the gap between the browser and Python worlds.
+
+### Pyodide
+
+<a href="https://pyodide.org/"><img src="../assets/images/pyodide.png"/></a>
+
+[Pyodide](https://pyodide.org/) is a version of the standard
+[CPython](https://python.org/) interpreter, patched to compile to WASM and
+work in the browser.
+
+It includes many useful features:
+
+* The installation of pure Python packages from [PyPI](https://pypi.org/) via
+  the [micropip](https://micropip.pyodide.org/en/stable/index.html) package
+  installer. Some packages with C extensions have versions compiled for WASM
+  and these can also be installed with `micropip`. There are plans afoot to
+  make WASM a target in PyPI so packages with C extenions can be automatically
+  compiled to WASM.
+* An active, friendly and technically outstanding team of volunteer
+  contributors (some of whom have been supported by the PyScript project).
+* Extensive official
+  [documentation](https://micropip.pyodide.org/en/stable/index.html), and many
+  tutorials found online.
+* Builds of Pyodide that include popular packages for data science like
+  [Numpy](https://numpy.org/), [Scipy](https://scipy.org/) and
+  [Pandas](https://pandas.pydata.org/).
+
+### MicroPython
+
+<a href="https://micropython.org/"><img src="../assets/images/micropython.png"/></a>
+
+[MicroPython](https://micropython.org/) is a lean and efficient implementation
+of the Python 3 programming language that includes a small subset of the Python
+standard library and is optimised to run on microcontrollers and in constrained
+environments (like the browser). 
+
+Everything needed to view a web page in a browser needs to be delivered
+over the network. The smaller the asset to be delivered can be, the better.
+MicroPython, when compressed for delivery to the browser, is only around
+170k in size - smaller than many images found on the web.
+
+This makes MicroPython particularly suited to browsers running in a more
+constrained environment such as on a mobile or tablet based device. Browsing
+with these devices usually uses (slower) mobile internet connections.
+Furthermore, because MicroPython is lean and efficient it still performs
+exceptionally well on these relatively underpowered devices.
+
+Thanks to collaboration between the MicroPython and PyScript projects, there is
+a foreign function interface for MicroPython. The MicroPython FFI deliberately
+copies the API of the FFI originally written for Pyodide - meaning it is
+relatively easy to migrate between the two supported interpreters.
+
+## Configuration
+
+Because PyScript is a browser based technology, the tab in which the web page
+is displayed is a very secure sandboxed computing environment for running your
+Python code. Think of it as similar to the notion of a
+[virtual environment](https://docs.python.org/3/tutorial/venv.html)
+in the Python world.
```

Comment:
this is just wrong. Virtualenvs are not secure nor sandboxd

Reply:
> Think of it as similar to the notion of...

In English, this is saying they're like each other, but not the same. That's literally what "similar" means. :-D

Reply:
Although I agree we should tell folks venvs are not secure or sandboxed.

Reply:
as a non-native speaker, what I understand is that virtualenvs are sandboxed :)
So, since we are talking about security, better to play safe and rewrite the sentence in a less ambiguous way :)

Reply:
I've just removed this since it's not really needed and will only cause confusion. :smile: